packages/terra-core-docs/src/terra-dev-site/doc/alert/AccessibilityGuide.2.doc.mdx

```diff
@@ -42,38 +43,103 @@ Terra Alert has an optional role prop but should be used carefully when the defa
 | Alert | See <Hyperlink href="https://w3c.github.io/aria/#alert" variant="external">WAI-ARIA Alert Role</Hyperlink>[<sup>[1]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region with important, and usually time-sensitive, information." |
 | Status | See <Hyperlink href="https://w3c.github.io/aria/#status" variant="external">WAI-ARIA Status Role</Hyperlink>[<sup>[1]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region whose content is advisory information for the user but is not important enough to justify an alert, often but not necessarily presented as a status bar." |
 
+#### Alert Notification Type
+
+<Notice variant="ux-recommendation" ariaLevel="6">
+
+It may be more appropriate to use a modal or component with `alertdialog` if presenting the user with an alert and required action to address the alert.
+
+</Notice>
+
+<ul>
+    <li>
+        The "alert" role/notification type will cause keyboard navigation focus to shift to the notification banner body.
+        This helps users quickly take action when it is critical but can optionally be disabled via the disableAlertActionFocus prop.
+    </li>
+    <li>
+        Notifications with "alert" role/notification type do not need to explicitly be in an aria-live region.
+        It is recommended to separate them from less critical notification banners to prevent assistive technologies from confusing them.
+    </li>
+    <li>
+        By default, Terra Alert uses the "alert" notification type.
+    </li>
+</ul>
+
+Demo of Terra Alert with an action element.
+
+<AlertActionFocusDemo />
+
 #### Assistive technologies
 
 In combination with the notification criticality, screen reader users should understand the importance of the notification when presented.
 
 * The `"alert"` role will interrupt the user and announce the high critical notification immediately.
 * The `"status"` role will not interrupt the user and present politely.
 
-<Notice variant="important" ariaLevel="5">
+#### Live regions
+
+<ul>
```

Comment:
Are these HTML-style blocks necessary outside of the Notice components? We should be able to create these lists using standard markdown, and maybe also enable the proper formatting when using backticks, etc.

Reply:
That's a good catch, I also checked with screen reader and it reads the list the same. Updated to using markdown here instead of html lists https://github.com/cerner/terra-core/pull/3815/commits/a11aabf2402e790b9b84d306ed15481dea26d354#diff-c525383d308e956417aad08bacff8ee0d585de12ccd7e0dc76c40f981b2a6d85R75